### PR TITLE
L2-705: SNS Remove Hotkey

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Method to remove neuron permissions.

# Changes

* New method in sns governance "removeNeuronPermissions".
* New method in sns wrapper "removeNeuronPermissions"

# Tests

* Test for new wrapper method.
* Test for new sns governance method.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker please review

File: packages/sns/src/governance.canister.ts [hunk 1]
```diff
@@ -102,6 +102,29 @@ export class SnsGovernanceCanister extends Canister<SnsGovernanceService> {
...
 
+  /**
+   * Remove permissions to a neuron for a specific principal
+   */
+  removeNeuronPermissions = async ({
+    neuronId,
+    permissions,
+    principal,
+  }: SnsNeuronPermissionsParams): Promise<void> => {
+    const request: ManageNeuron = {
```
Comment (peterpeterparker) [inline packages/sns/src/governance.canister.ts:113]: nth: extract a function `toCommand` or something - execpt the `command` I guess the `ManageNeuron` is the same for both add and remove?

Comment (peterpeterparker) [reply packages/sns/src/governance.canister.ts:113]: actually, maybe even refactor following

```
const response = await this.manageNeuron(request);
    this.assertManageNeuronError(response);
```

that's the same too I guess, so you can have a function that takes the `command`  as parameter

**Review by peterpeterparker — APPROVED** [review]
LGTM, added a comment about a nth refactoring to maybe avoid code duplication

File: packages/sns/src/governance.canister.ts (hunk 1 above)
Comment (lmuntaner) [reply packages/sns/src/governance.canister.ts:113]: ok.

Comment (lmuntaner) [reply packages/sns/src/governance.canister.ts]: done

**Pull request merged**